cylc/flow/task_state.py

```diff
@@ -416,6 +416,10 @@ def reset(
         """
         req = status
 
+        if forced and req in [TASK_STATUS_SUBMITTED, TASK_STATUS_RUNNING]:
+            # Forcing can only change completion status (there's no job).
```

Comment:
Somewhat confusing comment, `cylc set` can and does change the task status.

Reply:
👍 By "completion status" I meant task status corresponding to final task states, Bad wording, will fix!

Reply:
Done.